cf-deployment.yml

```diff
@@ -2846,9 +2847,9 @@ releases:
   version: 4.5.1
   sha1: b4d84a576f39c126656c4bc0116f5fbc228654f3
 - name: pxc
-  url: https://bosh.io/d/github.com/cloudfoundry-incubator/pxc-release?v=0.50.0
-  version: 0.50.0
-  sha1: 5ce091516a0c7a7d72e68b92865aaae85aab2d0d
+  version: 1.0.7
+  url: https://bosh.io/d/github.com/cloudfoundry/pxc-release?v=1.0.7
+  sha1: 3482ddfe955f7612a327ccb2a4261231db99d791
```

Comment:
Bumping release versions in the base manifest is somewhat involved as they need to be included in the ops-file that allows users to make use of pre-compiled releases. We have a [CI pipeline](https://concourse.wg-ard.ci.cloudfoundry.org/teams/main/pipelines/update-releases/jobs/update-pxc/builds/1) that normally takes care of all the heavy lifting automatically. However, I think that the change in GitHub organization from `cloudfoundry-incubator` to `cloudfoundry` has broken its ability to detect new versions. I've updated it to point to the new location, so it should detect and consume the new version shortly.

Once the release bump has made it to the `develop` branch, could you please remove this section and rebase your branch?